src/index.mjs

```diff
@@ -36,6 +42,25 @@ function isIgnored(node, filter) {
     : (filter.test || filter).call(filter, node.href, node);
 }
 
+/**
+ * Checks network conditions
+ * @param  {NetworkInformation}  conn    The connection information to be checked
+ * @return {Promise}  reject is the constrainst are met or resolved otherwise
```

Comment:
Return value documentation is no longer correct.

Reply:
Thanks, updated return value documentation.